conda/envs/dev-environment.yml

```diff
@@ -1,7 +1,7 @@
 name: dask-cudf-dev
 channels:
   - nvidia
-  - rapidsai
+  - rapidsai-nightly
```

Comment:
Sorry if this is a naive question, do we still need the `rapidsai` channel here?

Reply:
We should still have the `rapidsai` channel here with higher priority than the `rapidsai-nightly` channel.